src/common/timer/interaction-timer.js

```diff
@@ -6,8 +6,9 @@ import { isBrowserScope } from '../util/global-scope'
 export class InteractionTimer extends Timer {
   constructor (opts, ms) {
     super(opts, ms)
-    this.onRefresh = opts.onRefresh
     this.onPause = opts.onPause
+    this.onRefresh = opts.onRefresh
+    this.onResume = opts.onResume
```

Comment:
Maybe check if `opts.*` is a function or assign a `noop`.

```suggestion
    this.onPause = typeof opts.onPause === 'function' ? opts.onPause : () => { /* noop */ }
    this.onRefresh = typeof opts.onRefresh === 'function' ? opts.onRefresh : () => { /* noop */ }
    this.onResume = typeof opts.onResume === 'function' ? opts.onResume : () => { /* noop */ }
```